plat/allwinner/common/include/sunxi_helpers.h

```diff
@@ -8,6 +8,7 @@
 #define __SUNXI_HELPERS_H__
 
 uint16_t sunxi_read_soc_id(void);
+int platform_init_r_twi(uint16_t socid, bool use_i2c);
```

Comment:
Why does this function have a different prefix than all of the other sunxi shared functions?